src/test/java/nc/students/ayaz/controllers/UserControllerTest.java

```diff
@@ -65,6 +65,7 @@ public void shouldFetchUserWhenGetRequestSent() throws NoSuchUserException, IOEx
                 String.class
         );
 
+        System.out.println(response.getBody());
```

Comment:
&#x1F34E; **Issue**
You don't need to log anything inside your test. Test should be easy enough to understand without any logs.
You could add such statements during debugging, but after you are done you should remove them.